python/pyspark/pandas/namespace.py

```diff
@@ -2814,9 +2820,21 @@ def to_numeric(arg):
     1.0
     """
     if isinstance(arg, Series):
-        return arg._with_new_scol(arg.spark.column.cast("float"))
+        if errors == "coerce":
+            return arg._with_new_scol(arg.spark.column.cast("float"))
+        elif errors == "raise":
+            scol = arg.spark.column
+            scol_casted = scol.cast("float")
+            cond = F.when(
+                F.assert_true(~(scol.isNotNull() & scol_casted.isNull())).isNull(), scol_casted
```

Comment:
Can we simplify 

```
~(scol.isNotNull() & scol_casted.isNull())
```

to

```
scol.isNull() || casted_scol.isNonNull()
```

? See also https://www.dcode.fr/boolean-expressions-calculator

Reply:
Thanks for the link!